radicle-node/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Build
+FROM rust:1.65.0-slim@sha256:27349440d16ac15f52e8cae6181999a1b2fc05272d612a358d0e53078fcef88e as build
+
+RUN apt-get update && apt-get install -y pkg-config libssl-dev git cmake
```

Comment:
It's worth checking if it builds without `cmake` and `pkg-config` now

Reply:
This should be resolved now